lib/Resolver.js

```diff
@@ -0,0 +1,31 @@
+export default class NestedResolver {
+  constructor(resolvers) {
+    this.resolvers = resolvers.reverse();
```

Comment:
[`Array.prototype.reverse`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse) mutates array on which it was called 🙈 

```suggestion
    this.resolvers = resolvers.slice().reverse();
```

Reply:
Done via https://github.com/camunda/linting/commit/6b7b6ff2eadfc7666530813171fcea67edb38520